docs/pages/blog/mui-x-v8.md

```diff
@@ -0,0 +1,418 @@
+
+### Row spanning
+
+Row spanning has also been promoted to stable, providing a reliable and flexible solution for merging cells across multiple rows.
+This enhancement makes it easier to create complex, multi-row layouts within your Data Grid, allowing you to present your data more effectively.
+
+Learn how to set up and use [Row spanning](/x/react-data-grid/row-spanning/).
+
+### Server-side Aggregation and Lazy loading [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+
+The Data Source now supports advanced features such as server-side aggregation, on-demand data fetching for infinite scrolling, and lazy loading.
+
+Explore all your options when handling [Server-side data](/x/react-data-grid/server-side-data/).
+
+### Data source on the Community Plan
+
+Great news—the Data Source move from the Pro Plan to the Community Plan (MIT licensed)!
+
+We're unifying our approach to handling server-side data across all plans, creating a single, cohesive framework.
+
+Explore your options in our [Data source docs](/x/react-data-grid/server-side-data/).
+
+### New Toolbar
+
+The [redesigned toolbar](/x/react-data-grid/components/toolbar/) not only looks fantastic but also empowers developers one step further through an open composition of subcomponents, allowing you to tailor the toolbar's layout precisely to your application's needs.
+
+<figure>
+  <img width="1170" height="104" alt="The new toolbar's design" src="/static/blog/mui-x-v8/new-toolbar.png" loading="lazy" style="border: 0;">
+  <figcaption>The new toolbar's design</figcaption>
+</figure>
+
+This new developer experience, based on composition, may feel different from our previous approach—but it sets a new standard.
+We'll soon extend this enhanced experience to other key subcomponents, such as the filter panel and columns management panel, ensuring a cohesive and powerful toolkit for advanced customization.
+
+Dive into our new docs section on [Components usage](/x/react-data-grid/components/usage/).
+
+### Design agnostic core
+
+We understand customization is important to you.
+So with the ultimate goal of being fully customizable, we're taking the first step to allow using the Data Grid component without a single Material UI dependency (for example
+without Emotion), opening the possibility to use the Data Grid with any design system—and this is just the beginning.
+
+Next up, we'll work to provide the best experience for you to build a full-feature Data Grid that fits all your design requirements.
+
+### No columns overlay
+
+We've added a no-columns overlay to serve as a clear fallback when the Data Grid has no columns to show.
+
+Learn more in the [No columns overlay documentation](/x/react-data-grid/overlays/#no-columns-overlay).
+
+### Performance improvements
+
+We've made significant performance enhancements across the board—from smoother scrolling and faster Excel export to improved mount/resize and aggregation performance.
```

Comment:
To share benchmark or to remove. In its current form, removing this seems much better.